utils.go

```diff
@@ -211,6 +211,11 @@ func getRootlessStorageOpts(rootlessUID int) (StoreOptions, error) {
 	if path, err := exec.LookPath("fuse-overlayfs"); err == nil {
 		opts.GraphDriverName = "overlay"
 		opts.GraphDriverOptions = []string{fmt.Sprintf("overlay.mount_program=%s", path)}
+		for _, o := range systemOpts.GraphDriverOptions {
+			if strings.Contains(o, "ignore_chown_errors") {
+				opts.GraphDriverOptions = append(opts.GraphDriverOptions, o)
+			}
+		}
```

Comment:
Can there be multiple ignore_chow_errors that we'd capture?  if not, should we break the loop?

Reply:
It is breaking now.